src/components/CommonMessage/CommonMessage.scss

```diff
@@ -13,6 +13,9 @@
 		animation-duration: 0.2s;
 		animation-timing-function: ease-in-out;
 		animation-fill-mode: forwards;
+		&.small {
```

Comment:
see above comment, this should be _&--small {_ as it is a modifier class declaration.